modules/orchestrator/services/runtime/runtime.go

```diff
@@ -1698,11 +1704,11 @@ func (r *Runtime) FullGC() {
 			break
 		}
 		for i := range runtimes {
-			if runtimes[i].Workspace == string(apistructs.ProdWorkspace) {
-				r.fullGCForSingleRuntime(runtimes[i].ID, keep)
-			} else {
-				r.fullGCForSingleRuntime(runtimes[i].ID, 1)
+			keep, ok := rollbackCfg[runtimes[i].ProjectID][strings.ToUpper(runtimes[i].Workspace)]
+			if !ok {
+				keep = 5
 			}
```

Comment:
what if keep's value invalid? some like -1, 0 or much greater than 1000000

Reply:
nice review, it will change to between 0 and 100. done